app/views/hold/show.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= content_for(:title, "Thesis Submission | MIT Libraries") %>
+
+<h3 class="hd-3"><%= "Full hold history: #{@hold.thesis.title}" %></h3>
+<p><%= link_to 'Return to hold record', admin_hold_path(@hold) %></p>
+<% @hold.versions.reverse_each do |version| %>
+  <div class="box-content">
+    <h4 class="hd-4">Version <%= version.index + 1 %></h4>
+    <ul class="list-unbulleted">
+      <li>Event type: <%= version.event %></li>
+      <li>Modified by: <%= modified_by(version) %></li>
+      <li>
+        <table class="table table-simplified table-cozy">
+          <tbody>
+            <tr>
+              <th scope="col">Field name</th>
+              <th scope="col">Previous value</th>
+              <th scope="col">Current value</th>
```

Comment:
`Current value` may be misleading when you get into multiple changes to a Hold. Maybe something like `Previous value` and `Changed to value`? Or even `Old` and `New`? 🤷🏻

Reply:
Good catch! It's not really the "current value" unless it's the current version. Change forthcoming...